config/env.config.js

```diff
@@ -55,5 +57,77 @@ const config = {
     }
 };
 
+const RedirectPlugin = function(options) { 
```

Comment:
Actually, since this is being initialized as a plugin in the `webpack.common` file, I don't believe I can use lambda notation? That would not allow me to have a constructor.